rspace-bench/src/test/scala/coop/rchain/rspace/bench/RhoReplayBenchBaseState.scala

```diff
@@ -0,0 +1,30 @@
+package coop.rchain.rspace.bench
+
+import coop.rchain.catscontrib.TaskContrib._
+import monix.eval.Task
+import org.openjdk.jmh.annotations.{Level, Setup}
+
+abstract class RhoReplayBenchBaseState extends RhoBenchBaseState {
+
+  override def execute: Task[Vector[Throwable]] =
+    for {
+      result <- runTask
+      _      <- runtime.replaySpace.createCheckpoint()
+    } yield result
+
+  @Setup(value = Level.Iteration)
+  override def doSetup(): Unit = {
+    super.doSetup()
+
+    val randRun = rand
+    assert(createTest(term)(readErrors, runtime.reducer, randRun).unsafeRunSync.isEmpty)
+    (for {
+      setupCheckpoint <- runtime.space.createCheckpoint()
+      _               <- runtime.replaySpace.rigAndReset(setupCheckpoint.root, setupCheckpoint.log)
+      _ = assert(
+        createTest(setupTerm)(readErrors, runtime.replayReducer, randSetup).unsafeRunSync.isEmpty
+      )
+      _ = runTask = createTest(term)(readErrors, runtime.replayReducer, randRun)
```

Comment:
I know this code was merely moved, but why does `createTest` need to run twice? (see `assert` above)

Reply:
good point, I'll see what I can do about it

Reply:
the params are different ;)